nntrainer/layers/mol_attention_layer.cpp

```diff
@@ -37,7 +38,8 @@ void MoLAttentionLayer::forwarding(RunLayerContext &context, bool training) {
 void MoLAttentionLayer::calcDerivative(RunLayerContext &context) { /** NYI */ }
 
 void MoLAttentionLayer::setProperty(const std::vector<std::string> &values) {
-  /** NYI */
+  auto remain_props = loadProperties(values, mol_props);
```

Comment:
Could you also implement export_to as well?

Reply:
it coming in the new PR.